base/errorshow.jl

```diff
@@ -84,19 +84,8 @@ function showerror(io::IO, ex, bt; backtrace=true)
     end
 end
 
-function showerror(io::IO, ex::LoadError, bt; backtrace=true)
-    print(io, "LoadError: ")
-    showerror(io, ex.error, bt, backtrace=backtrace)
-    print(io, "\nin expression starting at $(ex.file):$(ex.line)")
-end
-showerror(io::IO, ex::LoadError) = showerror(io, ex, [])
-
-function showerror(io::IO, ex::InitError, bt; backtrace=true)
-    print(io, "InitError: ")
-    showerror(io, ex.error, bt, backtrace=backtrace)
-    print(io, "\nduring initialization of module ", ex.mod)
-end
-showerror(io::IO, ex::InitError) = showerror(io, ex, [])
+showerror(io::IO, ex::LoadError) = print(io, "LoadError in expression starting at ", ex.file, ":", ex.line)
```

Comment:
I think it would be clearer to say "Error while loading expression starting at ..."; `LoadError` is just a derived exception and not really a kind of error on its own. It also seems to be unnecessary to show the backtrace for a LoadError; I think it's always a prefix of the nested exception?

Also, it's nice to note that our backtrace machinery is now good enough that LoadError may be entirely unnecessary :tada:

Reply:
> I think it would be clearer to say "Error while loading expression starting at ..."

Sure.

> It also seems to be unnecessary to show the backtrace for a LoadError; I think it's always a prefix of the nested exception?

Yes. This isn't really specific to LoadError and @vtjnash had a good suggestion over at https://github.com/JuliaLang/julia/pull/29901#issuecomment-445314394.